release.config.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+module.exports = {
```

Comment:
this is neat. can you elaborate on the `channel` and `prerelease` flags? (just for me, no need to add a comment in the file)

Reply:
In this case I think semantic-release's beta docs will do a much better job at explaining this than any explanation I can offer. Open to discussing more IRL.

https://github.com/semantic-release/semantic-release/blob/2caafbaa2be54330b5b3e6dd71dda0270b566663/docs/recipes/pre-releases.md#publishing-pre-releases